ci-operator/step-registry/assisted/common/gather/assisted-common-gather-commands.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ assisted common gather command ************"
+
+# TODO: Remove once OpenShift CI will be upgraded to 4.2 (see https://access.redhat.com/articles/4859371)
+~/fix_uid.sh
+
+cat > run_test_playbook.yaml <<-EOF
+- name: Gather logs and debug information
+  hosts: all
+  vars:
+    LOGS_DIR: /tmp/artifacts
+    GATHER_CAPI_LOGS: "{{ lookup('env', 'GATHER_CAPI_LOGS') }}"
+    CLUSTER_GATHER:
+      - "{{ (lookup('env', 'SOSREPORT') == 'true') | ternary('--sosreport','', '') }}"
+      - "{{ (lookup('env', 'MUST_GATHER') == 'true') | ternary('--must-gather','', '') }}"
+      - "{{ (lookup('env', 'GATHER_ALL_CLUSTERS') == 'true') | ternary('--download-all','', '') }}"
+  tasks:
+    - name: Copy junit report files
+      copy:
+        remote_src: true
+        src: /home/assisted/reports
+        dest: "{{ LOGS_DIR }}"
+    - name: Run sos report
+      ansible.builtin.shell: |
+        source /root/config.sh
+        # Get sosreport including sar data
+        sos report --batch --tmp-dir {{ LOGS_DIR }} \
+          -o memory,container_log,filesys,kvm,libvirt,logs,networkmanager,networking,podman,processor,rpm,sar,virsh,yum \
+          -k podman.all -k podman.logs
+    - name: Copy libvirt qemu log
+      copy:
+        remote_src: true
+        src: /var/log/swtpm/libvirt/qemu/
+        dest: "{{ LOGS_DIR }}/libvirt-qemu"
+    - name: List swtpm-localca files to a file
+      ansible.builtin.shell: |
+        ls -ltr /var/lib/swtpm-localca/ >> {{ LOGS_DIR }}/libvirt-qemu/ls-swtpm-localca.txt
+    - name: Check minikube kubeconfig file existence
+      stat:
+        path: /root/.kube/config
+      register: kubeconfig
+    - name: Extract assisted service logs
+      ansible.builtin.shell: |
+        make download_service_logs
+      args:
+        chdir: /home/assisted
+      environment:
+        KUBECONFIG: "/root/.kube/config"
+      when: kubeconfig.stat.exists
+    - name: Extract capi logs
+      ansible.builtin.shell: |
+        make download_capi_logs
+      environment:
+        KUBECONFIG: "/root/.kube/config"
+      args:
+        chdir: /home/assisted
+      when: kubeconfig.stat.exists and GATHER_CAPI_LOGS == "true"
+    - debug:
+        msg: "CLUSTER_GATHER = {{ CLUSTER_GATHER }}"
+    - name: Download cluster logs
+      ansible.builtin.shell: |
+        source /root/config.sh
+        make download_cluster_logs
+      environment:
+        KUBECONFIG: "/root/.kube/config"
+        ADDITIONAL_PARAMS: "{{ CLUSTER_GATHER | join(' ') }}"
+      args:
+        chdir: /home/assisted
+    - name: Find all log files
+      find:
+        paths: /home/assisted/
+        patterns: "*.log"
+        recurse: true
+      register: log_files
+    - name: Print log file names
+      debug:
```

Comment:
is it just for debugging?
also for those other printings

Reply:
it's for debugging but i'm considering keeping it. WDYT?

Reply:
Sure no problem